kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/Aether.java

```diff
@@ -45,13 +45,16 @@
 import org.eclipse.aether.transport.file.FileTransporterFactory;
 import org.eclipse.aether.transport.http.HttpTransporterFactory;
 import org.eclipse.aether.transport.wagon.WagonProvider;
+import org.eclipse.aether.transport.wagon.WagonTransporterFactory;
 import org.eclipse.aether.util.repository.DefaultProxySelector;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
```

Comment:
i'm sorry for being annoying and picky, but can we avoid use this here? other than that.. it looks good to me

Reply:
Done